plugins/PrivacyManager/lang/en.json

```
@@ -246,6 +246,9 @@
         "ConsentManagerDetected": "%1$s consent manager was detected on your website. To learn about configuring Matomo to work with %1$s please read %2$sthis guide%3$s",
         "ConsentManagerConnected": "%1$s appears to already be configured to work with Matomo.",
         "UseRandomizeConfigId": "Randomize config_ID for enhanced privacy",
-        "RandomizeConfigIdNote": "By randomizing the config_ID for every request, this feature ensures that visitor tracking is compliant with the strictest privacy interpretations. This setting disables mechanisms that rely on user-agent data, cookies, or other identifiers that require explicit consent, allowing for anonymized tracking while respecting user privacy."
+        "RandomizeConfigIdNote": "By randomizing the config_ID for every request, this feature ensures that visitor tracking is compliant with the strictest privacy interpretations. This setting disables mechanisms that rely on user-agent data, cookies, or other identifiers that require explicit consent, allowing for anonymized tracking while respecting user privacy.",
+        "RandomizeConfigIdNoteWarning": "<strong>Please note: <\/strong>With this setting enabled, every action will be tracked as a new visit.",
```

```suggestion
        "RandomizeConfigIdNoteWarning": "%1$sPlease note:%2$s With this setting enabled, every action will be tracked as a new visit.",
```

I think we usually try to keep all HTML out of the translations where possible.

---

Can do, though I had a look and for html that wasn't dynamic like strong tags we had that already in translations, even in this very file. Will move it out.